torch/_dynamo/variables/tensor.py

```diff
@@ -871,25 +865,37 @@ class NumpyNdarrayVariable(TensorVariable):
     Use this for Tensor.numpy() call.
     """
 
+    @staticmethod
+    def create(tx, proxy, **options):
+        from .builder import wrap_fx_proxy_cls
+
+        return wrap_fx_proxy_cls(
+            target_cls=NumpyNdarrayVariable,
+            tx=tx,
+            proxy=proxy,
+            **options,
+        )
+
     def __init__(
         self,
-        proxy: torch.fx.Proxy,
+        proxy,
         **kwargs,
     ):
         super().__init__(proxy, **kwargs)
+        self.proxy = proxy
```

Comment:
What's going on here? Are there two proxies floating around now?

Reply:
We probably don't need this line, super already does that